src/neural-network.js

```diff
@@ -292,17 +292,10 @@ export default class NeuralNetwork {
    *       momentum: (number),
    *       activation: ['sigmoid', 'relu', 'leaky-relu', 'tanh']
    */
-  updateTrainingOptions(opts) {
```

Comment:
I originally had this method as private (i.e. starting with `_`) but then asked myself why.  I think it is fine to be exposed to the user but I could see it going either way.  Mostly if we don't have the `_` it indicates to the user that they are safe calling it without having strange side effects.

I would vote for keeping it public but it isn't a big deal.

Reply:
They set these options implicitly via the constructor or the `train`, do they need a third way to set the training options?

Reply:
no they don't need it, but if we have it already setup it is 0 cost to us letting it be explicitly exposed.  I am ok keeping it private I just thought I would bring it up.